Added information about translations provided by ibexa/i18n package (#1562)
ezsystems/developer-documentation: docs/guide/back_office_translations.md

# Back Office translations

## Enabling Back Office languages

> All translations are available as a part of the [[= product_name =]]. To enable Back Office translations, use the following configuration:

Reviewer: ```suggestion
All translations are available as a part of [[= product_name =]]. To enable Back Office translations, use the following configuration:
```

!!! tip

>     If you do want to add your own bundle with Back Office translation, you could install it manualy via composer.  

Reviewer: ```suggestion
    If you do want to add your own bundle with Back Office translations, you can install it manually via Composer.  
```

>     Also, you can manually add the necessary .xliff files to existing project.

Reviewer: ```suggestion
    Also, you can manually add the necessary .xliff files to an existing project.
```
